src/editor-previewer/editor-previewer.component.html

```diff
@@ -4,6 +4,9 @@
     <tab *ngFor="let preview of previews trackBy:trackByFunction" [heading]="preview.name">
       <div [ngSwitch]="preview.type">
         <div *ngSwitchCase="'html'" class="preview-container">
+          <template tabHeading>
+            <a target="_blank" [href]="preview.url">&#10138;</a>
```

Comment:
unfortunately, since `ng2-bootstrap` `tab` component creates an `anchor` so that's why we have `anchor` in `anchor` which is "forbidden" in HTML, and inner one doesn't work. So let's go back to `span` with `(click)` some style for mouse icon and link color (`#337ab7`).